index.md

```diff
@@ -1,6 +1,20 @@
 Hi! My name is David, and I am going to tell you what I have learned from my
 dad every day.
 
+## Saturday, May 11
+
+Today my dad told me  to down load a app called Jira. At first I was thinking,
```

Comment:
Two spaces between words, find these in vim with this command: `/ . ` then look for more occurences with this command: `n`